src/jit/lsra.h

```diff
@@ -1479,19 +1536,27 @@ class LinearScan : public LinearScanInterface
 #endif // FEATURE_PARTIAL_SIMD_CALLEE_SAVE
 
     //-----------------------------------------------------------------------
-    // TreeNodeInfo methods
+    // BuildNode methods
     //-----------------------------------------------------------------------
 
+    // The listNodePool is used to maintain the TreeNodeInfo for nodes that are "in flight"
+    // i.e. whose consuming node has not yet been handled.
+    LocationInfoListNodePool listNodePool;
+
     // The defList is used for the transient TreeNodeInfo that is computed by
-    // the TreeNodeInfoInit methods, and used in building RefPositions.
+    // the Build methods, and used in building RefPositions.
```

Comment:
Build =>BuildNode

Reply:
Only the main entry method is `BuildNode`. The others are `BuildXXX` where `XXX` is the type of name. So `Build` is the "generic" name for these, where `TreeNodeInfoInit` was before (I changed vanilla `TreeNodeInfoInit` to `BuildNode` because I thought, while, e.g. `BuildStoreLoc` was clear, `Build` alone wasn't). So I would prefer keep these comments as simply `Build`.